chapters/ch27.md

```diff
@@ -1,38 +1,37 @@
-->
+-> * ** Lựa chọn đặc trưng để giảm số lượng / kiểu đặc trưng đầu vào **: Kỹ thuật này có thể giúp giải quyết các vấn đề về phương sai, nhưng nó cũng có thể làm tăng độ chệch. Việc giảm một chút số lượng các đặc trưng (giả sử giảm từ 1.000 đặc trưng xuống còn 900 đặc trưng) dường như không có ảnh hưởng lớn đến độ chệch. Việc giảm đáng kể (giả sử giảm từ 1.000 đặc trưng xuống còn 100 đặc trưng, tức là giảm 10 lần) có nhiều khả năng có tác dụng đáng kể, miễn là bạn không loại trừ quá nhiều các đặc trưng hữu ích. Trong deep learning hiện đại, khi dữ liệu dồi dào, đã có những thay đổi từ việc lựa chọn đặc trưng và giờ đây hầu như chúng ta cung cấp tất cả các đặc trưng chúng ta có cho thuật toán và để thuật toán chọn ra những đặc trưng nào sẽ sử dụng dựa trên dữ liệu . Nhưng khi tập huấn luyện của bạn nhỏ, kỹ thuật lựa chọn đặc trưng vẫn có thể rất hữu ích.
 
 
 * **Decrease the model size** ​(such as number of neurons/layers): ​*Use with caution*.​ This technique could decrease variance, while possibly increasing bias. However, I don’t recommend this technique for addressing variance. Adding regularization usually gives better classification performance. The advantage of reducing the model size is reducing your computational cost and thus speeding up how quickly you can train models. If speeding up model training is useful, then by all means consider decreasing the model size. But if your goal is to reduce variance, and you are not concerned about the computational cost, consider adding regularization instead.
-->
+-> * ** Giảm kích thước mô hình ** (chẳng hạn như số lượng neurons / tầng): * Sử dụng một cách thận trọng *. Kỹ thuật này có thể làm giảm phương sai, trong khi có thể làm tăng độ chệch. Tuy nhiên, tôi không khuyến khích kỹ thuật này để giải quyết phương sai. Thêm regularization thường cho hiệu suất phân loại tốt hơn. Ưu điểm của việc giảm kích thước mô hình là giảm chi phí tính toán của bạn và do đó tăng tốc độ bạn huấn luyện mô hình. Nếu tăng tốc độ huấn luyện mô hình là hữu ích, thì bằng mọi cách hãy xem xét giảm kích thước mô hình. Nhưng nếu mục tiêu của bạn là giảm phương sai và bạn không quan tâm đến chi phí tính toán, thay vào đó, hãy xem xét việc thêm regularization.
```

Comment:
neurons / tầng --> neurons/tầng 
"hiệu suất" --> "hiệu quả" (cái này hình như a Tiệp có discuss & giải thích ở bản dịch 1 chapter nào đó, mình ko nhớ)

Reply:
Chỗ "* Sử dụng một cách thận trọng *" cần cho hai dấu "*" vào sát chữ. Hiện tại phần in nghiêng là ở phần sau.